app/components/overview-info-form/component.js

```diff
@@ -1,11 +1,13 @@
 import Ember from 'ember';
+import {translations} from '../../utils/translationStrings';
 import {validator, buildValidations} from 'ember-cp-validations';
 
 
 var range = function(start, stop) {
     var options = [];
     for (var i=start; i <= stop; i++) {
-        options.push(i);
+      var key = 'number' + i;
```

Comment:
It looks off in the diff above, but it looks fine here: https://github.com/samanehsan/isp/blob/09e7384b7b7895a1c159ead2c9a3853c82213ace/app/components/overview-info-form/component.js#L9 

Am I missing something?